recOrder/cli/apply_inverse_transfer_function.py

```diff
@@ -342,8 +290,7 @@ def apply_inverse_transfer_function_cli(
                 reconstructed_parameters_3d = inplane_oriented_thick_pol3d.apply_inverse_transfer_function(
                     tczyx_data[time_index],
                     intensity_to_stokes_matrix,
-                    wavelength,
-                    cyx_no_sample_data,
+                    cyx_no_sample_data=cyx_no_sample_data,
                     project_stokes_to_2d=False,
```

Comment:
(ndim==2)

Reply:
See comment above...feel free to reopen.